projects/spectator/test/pipe/alternating-sum.pipe.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { SpectatorPipe, createPipeFactory } from '@ngneat/spectator';
+
+import { StatsService } from './stats.service';
+import { AlternatingSumPipe } from './alternating-sum.pipe';
+
+describe('AlternatingSumPipe', () => {
+  let spectator: SpectatorPipe<AlternatingSumPipe>;
+  const createPipe = createPipeFactory({
+    pipe: AlternatingSumPipe,
+    template: `{{ prop | alternatingSum }}`
+  });
+
+  it('should compute the alternating sum of a given list of numbers (prop)', () => {
+    spectator = createPipe({
+      hostProps: {
+        prop: [1, 2, 3]
+      }
+    });
+    expect(spectator.element).toHaveText('2');
+  });
+
+  it('should compute the alternating sum of a given list of numbers (template override)', () => {
+    spectator = createPipe(`{{ [1, 2, 3] | alternatingSum }}`);
```

Comment:
Please use a different value (e.g. `[3, 4, 5]`) here so that we know that it uses this template.

Reply:
Sure, makes sense! 👍